src/consultations/consultations.service.ts

```diff
@@ -84,7 +84,28 @@ export class ConsultationsService {
     }))
   }
 
-  async findOne(id: number, user: UserEntity, participationId?: number) {
+  async findOne(id: number, user: UserEntity) {
```

Comment:
Emphasize the difference by specifying the return type: `Promise<ConsultationEntity>`